CONTRIBUTING.md

```diff
@@ -53,13 +53,18 @@ ESPnet2's recipes correspond to `egs2`. ESPnet2 applies a new paradigm without d
 For ESPnet2, we do not recommend preparing the recipe's stages for each corpus but using the common pipelines we provided in `asr.sh`, `tts.sh`, and 
 `enh.sh`. For details of creating ESPnet2 recipes, please refer to [egs2-readme](https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/README.md).
 
-The common pipeline of ESPnet2 recipes will take care of the `RESULTS.md` generation, model packing, and uploading. ESPnet2 models are maintained at Zenodo.
+The common pipeline of ESPnet2 recipes will take care of the `RESULTS.md` generation, model packing, and uploading. ESPnet2 models are maintained at Zenodo and Hugging Face.
 You can also refer to the document in https://github.com/espnet/espnet_model_zoo
 To upload your model, you need first:
 1. Sign up to Zenodo: https://zenodo.org/
 2. Create access token: https://zenodo.org/account/settings/applications/tokens/new/
 3. Set your environment: % export ACCESS_TOKEN="<your token>"
 
+To port models from zenodo using hugging face hub, 
+1. Create a hugging face account - https://huggingface.co/
+2. Request to be added to espnet organisation - https://huggingface.co/espnet
```

Comment:
I am not sure if it is required to request to be added to the ESPnet organization. After all, they can upload without being part of the organization. @sw005320 What do you think about this?

Reply:
I think this is useful to make it under the organization, but I agree that it does not have to be a requirement. 
I could read that this description is more-like a suggestion instead of a requirement, and it is OK for me.